### PR TITLE
Replenish effective mode in iotdb-system.properties

### PR DESCRIPTION
**Pull request by shuwenwei**
## Description
1. Replenish effective mode in iotdb-system.properties
2. Remove '\r' in iotdb-system.properties

### CODE REVIEW

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

File: iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/conf/ConfigurationFileUtils.java [hunk 1]
```diff
@@ -158,7 +160,7 @@ public static String readConfigurationTemplateFile() throws IOException {
         BufferedReader reader = new BufferedReader(isr)) {
       String line;
       while ((line = reader.readLine()) != null) {
-        content.append(line).append(System.lineSeparator());
+        content.append(line).append("\n");
```
[the remark this answers is not in the document]
Comment (shuwenwei) [reply iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/conf/ConfigurationFileUtils.java:163]: In order to be compatible with some scripts, the newline characters in the configuration file need to be used uniformly '\n'

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/utils/TypeInferenceUtils.java [hunk 2]
```diff
@@ -40,6 +41,7 @@
 
 public class TypeInferenceUtils {
 
+  private static final IoTDBConfig CONF = IoTDBDescriptor.getInstance().getConfig();
   private static final TSDataType booleanStringInferType =
       IoTDBDescriptor.getInstance().getConfig().getBooleanStringInferType();
```
Comment (HTHou) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/utils/TypeInferenceUtils.java:46]: It can be removed.

Comment (HTHou) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/utils/TypeInferenceUtils.java:46]: It can be removed.

**Pull request merged**